### DISCUSSION TITLE
Synchronizing LL Desktop

### DISCUSSION THREAD

**Discussion by prometx** [LedgerHQ/ledger-live-desktop#3432] [Q&A]
Hi, i had an incorrect $xlm balance in "accounts" (an amount which showed on chain as having transacted, but was not received and updated to my ledger stellar wallet).

I cleared the cache to resync, but it has been going for some time (~3hours?), and I just wondered a few things:

1) is three hours "long, short, or middling" for this process in your experience?

2) once LL auto-locks its interface, does sync "pause", or will an initiated sync process proceed in the background once the app has exited to the login screen?

3) i am using the linux desktop app, just updated to newest version a few days ago (v2.19.0). is there a way i can monitor this process or get some feedback via the LL process in linux? i.e., can I check if it has "stalled"?

4) would closing the app and restarting it have a potentially good, potentially bad, or 
indifferent effect on the sync process?

thanks!

**Comment by cthepot** [comment]
Hi @prometx,

1. no it should take a few seconds only, I'd suggest to check your network and upgrade to the latest version. you can check with support for assistance https://support.ledger.com/
2. & 3 yes but you should check on a Linux's activity monitor tool (I'm not a Linux user myself)
4. It should not have a negative impact, Synchronization happens every couple of minutes or so.

Hope I answered your concern,
